org.aposin.licensescout.parent/pom.xml

```diff
@@ -1,242 +1,255 @@
-					<groupId>org.jacoco</groupId>
-					<artifactId>jacoco-maven-plugin</artifactId>
-					<version>${jacoco.version}</version>
-				</plugin>
-	
-				<!-- groupId: org.eclipse.m2e -->
-				<!--This plugin's configuration is used to store Eclipse m2e settings 
-					only. It has no influence on the Maven build itself. -->
-				<plugin>
-					<groupId>org.eclipse.m2e</groupId>
-					<artifactId>lifecycle-mapping</artifactId>
-					<version>1.0.0</version>
-					<configuration>
-						<lifecycleMappingMetadata>
-							<pluginExecutions>
-								<pluginExecution>
-									<pluginExecutionFilter>
-										<groupId>
-											org.apache.maven.plugins
-										</groupId>
-										<artifactId>
-											maven-plugin-plugin
-										</artifactId>
-										<versionRange>[3.2,)</versionRange>
-										<goals>
-											<goal>descriptor</goal>
-											<goal>helpmojo</goal>
-										</goals>
-									</pluginExecutionFilter>
-									<action>
-										<ignore></ignore>
-									</action>
-								</pluginExecution>
-							</pluginExecutions>
-						</lifecycleMappingMetadata>
-					</configuration>
-				</plugin>
-
-			</plugins>
-
-		</pluginManagement>
-	</build>
-
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.aposin.licensescout</groupId>
+	<artifactId>org.aposin.licensescout.parent</artifactId>
+	<version>1.3.1-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+	<name>LicenseScout parent</name>
+	<description>Common definitions for projects of LicenseScout</description>
+
+	<url>http://www.aposin.org</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!-- Dependencies version definitions -->
+		<commons-lang3.version>3.8.1</commons-lang3.version>
+		<velocity.version>2.0</velocity.version>
+		<commons-collections4.version>4.2</commons-collections4.version>
+		<commons-io.version>2.6</commons-io.version>
+		<maven-assembly-plugin.version>3.1.1</maven-assembly-plugin.version>
+		<maven-model.version>3.6.0</maven-model.version>
+		<maven-plugin-api.version>3.6.0</maven-plugin-api.version>
+		<maven-plugin-annotations.version>3.6.0</maven-plugin-annotations.version>
+		<maven-plugin-plugin.version>3.6.0</maven-plugin-plugin.version>
+		<maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
+		<plexus-utils.version>3.1.1</plexus-utils.version>
+		<junit.version>4.12</junit.version>
+		<json.version>20180813</json.version>
+
+		<!-- other dependencies versions -->
+
+		<mariadb-java-client.version>2.4.1</mariadb-java-client.version>
+
+		<!-- Maven plugin definitions -->
+		<maven-invoker-plugin.version>3.0.1</maven-invoker-plugin.version>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>

```

Comment:
Do these properties (maven.compiler target/source) need to be updated too?

Reply:
Yes, fixed this with the new push.